tests/functional/aws-node-sdk/test/object/websiteGet.js

```diff
@@ -565,5 +565,52 @@ describe('User visits bucket website endpoint', () => {
                 }, done);
             });
         });
+
+        describe('with bucket policy', () => {
+            beforeEach(done => {
+                const webConfig = new WebsiteConfigTester('index.html');
+                s3.putBucketWebsite({ Bucket: bucket,
+                    WebsiteConfiguration: webConfig }, err => {
+                    assert.strictEqual(err,
+                        null, `Found unexpected err ${err}`);
+                    s3.putBucketPolicy({ Bucket: bucket, Policy: JSON.stringify(
+                        {
+                            Version: '2012-10-17',
+                            Statement: [{
+                                Sid: 'PublicReadGetObject',
+                                Effect: 'Allow',
+                                Principal: '*',
+                                Action: ['s3:GetObject'],
+                                Resource: [`arn:aws:s3:::${bucket}/*`],
```

Comment:
I was thinking of having the actual object name here, instead of the wildcard, to make sure we test the bugfix, otherwise I feel like this is not testing the changes (but the test is welcome anyway)

Reply:
I've just replaced it with the index

Reply:
Thanks, LGTM :slightly_smiling_face: